src/test/java/io/vlingo/http/resource/ServerTest.java

```diff
@@ -179,7 +178,7 @@ public void setUp() throws Exception {
 
     consumer = world.actorFor(ResponseChannelConsumer.class, Definition.has(TestResponseChannelConsumer.class, Definition.parameters(progress)));
 
-    client = new BasicClientRequestResponseChannel(Address.from(Host.of("localhost"), serverPort, AddressType.NONE), consumer, 100, 10240, world.defaultLogger());
+    client = new NettyClientRequestResponseChannel(Address.from(Host.of("localhost"), serverPort, AddressType.NONE), consumer, 100, 10240);
```

Comment:
@alexguzun I didn't notice before, but this doesn't use the World logger. Is there a specific reason?

Reply:
For the Netty client implementation, I have decided, as previously discussed when talking about the issues of the current logging implementation, to use the SLF4J logger directly:

See  `io.vlingo.wire.fdx.bidirectional.netty.client.NettyClientRequestResponseChannel`
```
private final static Logger logger = LoggerFactory.getLogger(NettyClientRequestResponseChannel.class)
```

This allows configuring the the logging level specifically for the client:
` <logger name="io.vlingo.wire.fdx.bidirectional.netty.client" level="DEBUG"/>`

Also using the SLF4J logger directly captures the exact timestamp of the events, which is crucial when debugging issues.

Reply:
Ok, sorry I didn't remember that. Approving.